src/core/packages/chrome/browser-internal/src/project_navigation/project_navigation_service.ts

```diff
@@ -397,7 +403,7 @@ export class ProjectNavigationService {
   }
 
   private setProjectHome(homeHref: string) {
-    this.projectHome$.next(homeHref);
+    this.projectHome$.next(this.uiSettings?.get('defaultRoute') || homeHref);
```

Comment:
another option then is to make it default on getProjectHome$ level

getProjectHome$: () => {
        return this.projectHome$.pipe(map(home => home || this.uiSettings?.get('defaultRoute')));
      },


I think it would be better because now if setProjectHome never called then we never set the default